### PR TITLE
Compat updates for String rename

### PR DESCRIPTION
**Pull request by StefanKarpinski**

### CODE REVIEW

**Comment by yuyichao** [comment]
This one looks reasonably private so the change is probably OK.

File: REQUIRE [hunk 1]
```diff
@@ -2,3 +2,4 @@ julia 0.4
 HttpCommon
 HttpParser
 MbedTLS
+Compat
```
Comment (tkelman) [inline REQUIRE]: add minimum version

**Comment by andreasnoack** [comment]
@malmaud Can this one get merged?

**Comment by malmaud** [comment]
Yes

**Pull request merged**